### PR TITLE
Update to Secp Keys

### PR DESCRIPTION
**Pull request by wip-abramson**

### CODE REVIEW

**Comment by wip-abramson** [comment]
Okay, @jandrieu I believe this is the reorg that we discussed. Let me know if anything looks off.

**Review by jandrieu — CHANGES_REQUESTED** [review]
Unfortunately, did doc is an unfortunate and outdated term.

I'd like to switch all of these to just "did". So the folder is "dids" and each file has "did" in it instead of "diddoc", e.g., `certifier.did.json`

Also, "projects" should be "credentials", which will contain all the VCs we need for the example. The project proposal is just one of them.

**Comment by wip-abramson** [comment]
Okay @jandrieu this should be good to go now. I also updated the Readme to reflect the new artifacts and file structure.

In the future I will look to add tutorial for how to generate own artifacts using didkit.

File: README.md [hunk 1]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
...
 Many of the identifiers used in this set of artifacts are DIDs based on cryptographic public/private keypairs. This allows the artifacts' identifiers to be created offline and secured using cryptographic signatures.
 
-The public/private key pairs for these identifiers have been generated using Ed25519:
+The public/private key pairs are Secp256k1 keys generated using [didkit](https://spruceid.dev/docs/didkit/) and represented in [Json Web Key](https://datatracker.ietf.org/doc/html/rfc7517) format. The .jwk files contain both the public and private keys for each actor. From these keys a DID and associated DID Document has been generated using didkit.
 
 * **HydroElec** &mdash; The producer behind the proposed project.
-  * private Key [./rec/hydroelec.ed25519](./rec/hydroelec.ed25519)
-  * public key [./rec/hydroelec.ed25519.pub](./rec/hydroelec.ed25519.pub)
+  * Key pair [./rec/keys/hydroelec.jwk](./rec/keys/hydroelec.jwk)
+  * DID [./rec/dids/hydroelec.did.json](./rec/dids/hydroelec.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
  * DID document [./rec/dids/hydroelec.did.json](./rec/dids/hydroelec.did.json)
```

File: README.md [hunk 2]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
...
 
 * **Smart Meter 1** &mdash; The first of two electric meters that issue signed VCs as evidence of electricity production.
-  * private Key [./rec/meter1.ed25519](./rec/meter1.ed25519)
-  * public key [./rec/meter1.ed25519.pub](./rec/meter1.ed25519.pub)
+  * Key pair [./rec/keys/meter1.jwk](./rec/keys/meter1.jwk)
+  * DID [./rec/dids/meter1.did.json](./rec/dids/meter1.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
  * DID document [./rec/dids/meter1.did.json](./rec/dids/meter1.did.json)
```

File: README.md [hunk 3]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
...
 
 * **Smart Meter 2** &mdash; The second of two electric meters that issue signed VCs as evidence of electricity production.
-  * private Key [./rec/meter2.ed25519](./rec/meter2.ed25519)
-  * public key [./rec/meter2.ed25519.pub](./rec/meter2.ed25519.pub)
+  * Key pair [./rec/keys/meter2.jwk](./rec/keys/meter2.jwk)
+  * DID [./rec/dids/meter2.did.json](./rec/dids/meter2.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
  * DID document [./rec/dids/meter2.did.json](./rec/dids/meter2.did.json)
```

File: README.md [hunk 4]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
...
 
 * **UN FCCC** &mdash; The issuing authority for REC2022 Renewable Energy Certificates.
-  * private Key [./rec/unfccc.ed25519](./rec/unfccc.ed25519)
-  * public key [./rec/unfccc.ed25519.pub](./rec/unfccc.ed25519.pub)
+  * Key pair [./rec/keys/unfccc.jwl](./rec/keys/unfccc.jwk)
+  * DID [./rec/dids/unfccc.did.json](./rec/dids/unfccc.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
  * DID document [./rec/dids/unfccc.did.json](./rec/dids/unfccc.did.json)
```

File: README.md [hunk 5]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
...
-  * private Key [./rec/un.rec2022.ed25519](./rec/un.rec2022.ed25519)
-  * public key [./rec/un.rec2022.ed25519.pub](./rec/un.rec2022.ed25519.pub)
+  * Key pair [./rec/keys/un.rec2022.jwk](./rec/keys/un.rec2022.jwk)
+  * DID [./rec/dids/un.rec2022.did.json](./rec/dids/un.rec2022.did.json)
 
 * **Certifier** &mdash; A UN authorized project certifier. They review the project proposal from the producer and issue an REC2022 Certification verifiable credential.
-  * private Key [./rec/certifier.ed25519](./rec/certifier.ed25519)
-  * public key [./rec/certifier.ed25519.pub](./rec/certifier.ed25519.pub)
+  * Key pair [./rec/keys/certifier.jwk](./rec/keys/certifier.jwk)
+  * DID [./rec/dids/certifier.did.json](./rec/dids/certifier.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
  * DID document [./rec/dids/certifier.did.json](./rec/dids/certifier.did.json)
```

File: README.md [hunk 6]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
...
 
 * **Verifier** &mdash; A UN authorized project verifier. They review the project on-site and issue an REC2022 Verification verifiable credential, based the propsoal, the REC2022 Certification, and the evidence produced by the Project's smart meters.
-  * private Key [./rec/verifier.ed25519](./rec/verifier.ed25519)
-  * public key [./rec/verifier.ed25519.pub](./rec/verifier.ed25519.pub)
+  * Key pair [./rec/keys/verifier.jwk](./rec/keys/verifier.jwk)
+  * DID [./rec/dids/verifier.did.json](./rec/dids/verifier.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
  * DID document [./rec/dids/verifier.did.json](./rec/dids/verifier.did.json)
```

File: README.md [hunk 7]
```diff
@@ -66,11 +66,11 @@ Throughout these examples, we use a fictional did:example which uses an Ed25519
...
 #### 1. HydroElec
 
-The included files, [./rec/hydroelec.ed25519](./rec/hydroelec.ed25519) and [./rec/hydroelec.ed25519.pub](./rec/hydroelec.ed25519.pub) are the private and public keys, respectively, that control this DID:
+The file [./rec/keys/hydroelec.jwk](./rec/hydroelec.ed25519) is a representation of the private and public keys, respectively, that control this DID:
 
-```did:example:AAAAC3NzaC1lZDI1NTE5AAAAICwtd+9EJPeiBDh37Ffi6uJ6grYqtH97WrJW0fyui9rX```
+```did:key:zQ3shPm1156YHd3pzvhz8BMtZPT9LnTxtGyFBnm8tzJmjnBWD```
 
-See [./rec/hydroelec.did.json](./rec/hydroelec.did.json) for the DID Document for HydroElec.
+When this DID is resolved it returns the DID Document found in [./rec/dids/hydroelec.did.json](./rec/dids/hydroelec.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
When this DID is resolved it returns the DID document found in [./rec/dids/hydroelec.did.json](./rec/dids/hydroelec.did.json)
```

File: README.md [hunk 8]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
...
-  * private Key [./rec/unfccc.ed25519](./rec/unfccc.ed25519)
-  * public key [./rec/unfccc.ed25519.pub](./rec/unfccc.ed25519.pub)
+  * Key pair [./rec/keys/unfccc.jwl](./rec/keys/unfccc.jwk)
+  * DID [./rec/dids/unfccc.did.json](./rec/dids/unfccc.did.json)
 
 * **UN Rec2022** (DOES THIS NEED TO BE CRYPTOGRAPHIC?)
-  * private Key [./rec/un.rec2022.ed25519](./rec/un.rec2022.ed25519)
-  * public key [./rec/un.rec2022.ed25519.pub](./rec/un.rec2022.ed25519.pub)
+  * Key pair [./rec/keys/un.rec2022.jwk](./rec/keys/un.rec2022.jwk)
+  * DID [./rec/dids/un.rec2022.did.json](./rec/dids/un.rec2022.did.json)
```
Comment (jandrieu) [inline README.md]:
```suggestion
  * DID document [./rec/dids/un.rec2022.did.json](./rec/dids/un.rec2022.did.json)
```

**Review by jandrieu — CHANGES_REQUESTED** [review]
add "document" to did document references

File: README.md [hunk 9]
```diff
@@ -10,42 +10,42 @@ These artifacts help realize the Renewable Energy Certificates Focal Use Case.
 
 Many of the identifiers used in this set of artifacts are DIDs based on cryptographic public/private keypairs. This allows the artifacts' identifiers to be created offline and secured using cryptographic signatures.
 
-The public/private key pairs for these identifiers have been generated using Ed25519:
+The public/private key pairs are Secp256k1 keys generated using [didkit](https://spruceid.dev/docs/didkit/) and represented in [Json Web Key](https://datatracker.ietf.org/doc/html/rfc7517) format. The .jwk files contain both the public and private keys for each actor. From these keys a DID and associated DID Document has been generated using didkit.
```
Comment (wip-abramson) [inline README.md]:
```suggestion
The public/private key pairs are Secp256k1 keys generated using [didkit](https://spruceid.dev/docs/didkit/) and represented in [Json Web Key](https://datatracker.ietf.org/doc/html/rfc7517) format. The .jwk files contain both the public and private keys for each actor. From these keys a DID and associated DID document has been generated using didkit.
```

**Review by jandrieu — APPROVED** [review]

**Pull request merged**